gotpm/cmd/seal.go

```diff
@@ -102,7 +110,14 @@ Thus, algorithm and PCR options are not needed for the unseal command.`,
 		defer srk.Close()
 
 		fmt.Fprintln(debugOutput(), "Unsealing data")
-		secret, err := srk.Unseal(&sealed)
+
+		var secret []byte
+		if certify {
+			secret, err = srk.Unseal(&sealed, tpm2tools.ExpectedPCRs{Pcrs: sealed.CertifiedPcrs})
```

Comment:
I'm concerned that as `sealed` is attacker controlled data, this might be vulnerable to a downgrade attack.

Reply:
If the attacker tamper with the CertifiedPcrs, or remove them all together, the certification should fail.